tests/models/test_tpu.py

```diff
@@ -135,7 +139,8 @@ def test_model_16bit_tpu_index(tmpdir, tpu_core):
         precision=16,
         enable_progress_bar=False,
         max_epochs=2,
-        tpu_cores=[tpu_core],
+        accelerator="tpu",
+        devices=[tpu_core],
```

Comment:
Rename the var name to `devices` for consistency?

Reply:
I think it should pass the tests...